Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -3619,7 +3619,7 @@ public void run() {
 	 * @see BranchError	
 	 */
 	public interface BranchListResponseListener {
-		public void onReceivingResponse(JSONArray list, BranchError error);
+		void onReceivingResponse(JSONArray list, BranchError error);
```

Comment:
Haha, no worries. I think I'm the only remote currently. As I said, if we're not, we certainly should be. Perhaps we can get lint added to our Circle CI! @sojanpr is indeed the Android master now, so he'll have to lead this up. Just dropping another item on his plate :wink: 

Thanks for all these PRs, they hit a lot of low hanging fruit that we should be taking care of. I'll let Sojan do most of the actual review here, but it seems like most of these are pretty much just drop in fixes :+1: